internal/ansible/events/log_events.go

```diff
@@ -62,22 +64,50 @@ func (l loggingEventHandler) Handle(ident string, u *unstructured.Unstructured,
 		"job", ident,
 	)
 
+	// Parse verbosity from CR
+	verbosityAnnotation := 0
+	if annot, exists := u.UnstructuredContent()["metadata"].(map[string]interface{})["annotations"]; exists {
```

Comment:
I think these are unsafe casts, we'd need something more like:

```
if metadata, ok := u.UnstructuredContent()["metadata"].(map[string]interface{}); ok {
  if annot, exists := metadata["annotations"]; exists {
...
```

otherwise we could get panics here

Reply:
The metadata field will exist in UnstructuredContent(), so this checks whether or not the annotations field exists.
Also, without the ``.(map[string]interface{})`` assertion, it'll error with ``invalid operation: u.UnstructuredContent()["metadata"]["annotations"] (type interface {} does not support indexing)``, so it is necessary.